app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+import { USER_AVATAR_URL } from "../constants";
+import { requiredDateFormat, relativeCreationTime } from "../utils";
+
+const CardDiv = ({ item }) => (
+  <div className="m-3 w-full  border border-gray-200 p-4 shadow-sm  dark:border-gray-500 dark:bg-gray-600">
+    <Header
+      title={item.title}
+      actionBlock={
+        <Dropdown buttonStyle="text" icon={MenuVertical}>
+          <li>Edit</li>
+          <li>Delete</li>
```

Comment:
Thoroughly go through the Slack thread where I have put up a bunch of review comments and fix the issue with this piece of code.

Reply:
Is usage of buttons a better practice than simple list elements? Please do correct me if I'm wrong.

Reply:
We should try to stick onto semantic HTML since it helps with maintaining accessibility standards of the web page. Refer: https://github.com/bigbinary/neeto-ui/issues/1327#issuecomment-1253459534

Reply:
Thank you. Will make the necessary changes.